jenkins/osa_onmetal.functions.groovy

```diff
@@ -205,7 +205,51 @@ def delete_onmetal(datacenter_tag) {
 
 }
 
+def setup_during_test(host_ip) {
+    
+    sh """
+    scp -o StrictHostKeyChecking=no -r /home/ubuntu/workspace/Joshs_sandbox@3/during-upgrade-tests root@${host_ip}:/root 
```

Comment:
Why are you using this hardcoded path that is referring to a local jenkins workspace? Joshs_sandbox@3